orchestration-vs-choreography/README.md

```diff
@@ -0,0 +1,7 @@
+# Orchestration vs. Choreography: The good, the bad, and the trade-offs
+
+## Abstract
+
+If you’ve spent time building or maintaining event-driven systems, you may have bumped into the terms orchestration and choreography. While the web offers a vast range of articles that carefully outline the pros and cons of each pattern, the real challenge is left to the reader: deciphering which pattern best fits your specific use case by understanding which arguments don’t apply to your use case and weighing only the relevant ones in your decision.
```

Comment:
"If you've spent time..." may start off on the wrong foot. What if I haven't, and I'm curious about the design approach?

Punched up abstract option:

An event-driven system can provide your organization benefits, such as decoupling consumers and producers, improved resilience, and more reliable data models. But first, you must understand the concepts used to build these complex architectures. 

Central to event-driven system design are the concepts of **orchestration** and **choreography**. Picking between the two concepts is challenging, with unique use cases making it unclear what pattern best fits. An unclear understanding can lead to analysis paralysis, ultimately keeping you from taking the exciting leap into event-driven systems.

In this session, **Event-driven systems expert Laila Bougria** will guide you through **orchestration** and **choreography**, all while building a mental framework of pros and cons to inform your decisions. With your new-found knowledge, you can turn what were once abstract concepts into tangible decision-making frameworks to steer your organization in the right direction.

Reply:
Thanks for the feedback @khalidabuhakmeh 🙏🏼 
Rachel, Jodie and me did a round on it today and perfected it a bit further, I appreciate all your help!